easybuild/easyconfigs/h/HPL/HPL-2.1-pomkl-2016.03.eb

```diff
@@ -0,0 +1,23 @@
+name = 'HPL'
+version = '2.1'
+
+homepage = 'http://www.netlib.org/benchmark/hpl/'
+description = """HPL is a software package that solves a (random) dense linear system in double precision (64 bits) arithmetic 
+ on distributed-memory computers. It can thus be regarded as a portable as well as freely available implementation of the 
+ High Performance Computing Linpack Benchmark."""
+
+toolchain = {'name': 'pomkl', 'version': '2016.03'}
+toolchainopts = {'optarch': True, 'usempi': True}
+
+sources = [SOURCELOWER_TAR_GZ]
+source_urls = ['http://www.netlib.org/benchmark/%(namelower)s']
+
+# fix Make dependencies, so parallel build also works
+patches = ['HPL_parallel-make.patch']
+
+# needs to include -ldl or else we get
+# .../libmkl_core.a(mkl_memory_patched.o): undefined reference to symbol 'dlsym@@GLIBC_2.2.5'
+import os
+os.environ['LIBS'] = '-ldl'
```

Comment:
ah, crap, only noticed this now...

this is a hack at best, no @bartoldeman?

it actually indicates that something is missing in the toolchain support somewhere?

@geimer is this the problem you mentioned during today's conf call?